pkg/model/iam/iam_builder.go

```diff
@@ -310,6 +310,10 @@ func (r *NodeRoleNode) BuildAWSPolicy(b *PolicyBuilder) (*Policy, error) {
 		addLyftVPCPermissions(p, resource, b.Cluster.Spec.IAM.Legacy, b.Cluster.GetName())
 	}
 
+	if b.Cluster.Spec.Networking != nil && b.Cluster.Spec.Networking.Calico != nil && b.Cluster.Spec.Networking.Calico.AwsSrcDstCheck != "" {
```

Comment:
Should this be done on Masters also?

Reply:
yes, should be done on masters as well, adding

Reply:
Likewise here, shouldn't we check to see if the "AwsSrcDstCheck" field is not "DoNothing?"